programs/ziti-edge-tunnel/netif_driver/linux/tun.c

```diff
@@ -130,12 +144,56 @@ static int run_command(const char *cmd, ...) {
     return r;
 }
 
-static int run_command_ex(bool log_nonzero_ec, const char *cmd, ...) {
-    va_list args;
-    va_start(args, cmd);
-    int r = run_command_va(log_nonzero_ec, cmd, args);
-    va_end(args);
-    return r;
+static int is_systemd_resolved_acquired_name(sd_bus *bus) {
+    int r;
+    _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+    _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+    char** acquired = NULL;
+
```

Comment:
total nit: i notice quite a few "double returns" as whitespace separation of code. i'd personally just use one line... not a huge deal